### PR TITLE
Leave a Status Community

### PR DESCRIPTION
**Pull request by cheny0**

### CODE REVIEW

**Review by jorge-campo — CHANGES_REQUESTED** [review]
Thanks for your work on this, @cheny0 – Here are my review comments:

- In some cases, people may want to leave a community because they receive too many notifications. Your article can improve with a tip-style admonition, right after your intro sentence, telling users that they can configure their community notifications if they want to leave because of notification fatigue. And here, you can link to the future 'Configure community and channel notifications' article.
- In your 'What to expect' section, the sentences are somehow incomplete. Yes, this content means what to expect _when you leave a community_, but it's nevertheless helpful to remind the reader of that. Having said so, I'd rephrase your sentences a bit:
    - You no longer receive messages from the community. -> _When you leave a community, you stop receiving messages or updates from that community_.
    - Status does not notify the community owner and other members of your action. -> Status does not notify the community owner and other members _when you leave_.
    - You can join the community again. -> You can _request to_ join the community again. _Depending on the community rules, rejoining might require a new approval process or specific tokens._

**Comment by cheny0** [comment]
> Thanks for your work on this, @cheny0 – Here are my review comments:
> 
>     * In some cases, people may want to leave a community because they receive too many notifications. Your article can improve with a tip-style admonition, right after your intro sentence, telling users that they can configure their community notifications if they want to leave because of notification fatigue. And here, you can link to the future 'Configure community and channel notifications' article.
> 
>     * In your 'What to expect' section, the sentences are somehow incomplete. Yes, this content means what to expect _when you leave a community_, but it's nevertheless helpful to remind the reader of that. Having said so, I'd rephrase your sentences a bit:
>       
>       * You no longer receive messages from the community. -> _When you leave a community, you stop receiving messages or updates from that community_.
>       * Status does not notify the community owner and other members of your action. -> Status does not notify the community owner and other members _when you leave_.
>       * You can join the community again. -> You can _request to_ join the community again. _Depending on the community rules, rejoining might require a new approval process or specific tokens._

Thanks @jorge-campo! I have applied your comments and merged the content.
